config/features/sitenow_intranet/search_api.index.sitenow_intranet_search_index.yml

```diff
@@ -3,51 +3,52 @@ langcode: en
 status: true
 dependencies:
   config:
-    - field.storage.node.body
     - search_api.server.sitenow_intranet_search_server
   module:
-    - node
     - search_api
+    - node
 id: sitenow_intranet_search_index
 name: 'Sitenow Intranet Search Index'
 description: ''
 read_only: false
 field_settings:
-  body:
-    label: Body
-    datasource_id: 'entity:node'
-    property_path: body
-    type: text
-    dependencies:
-      config:
-        - field.storage.node.body
-  title:
-    label: Title
-    datasource_id: 'entity:node'
-    property_path: title
+  rendered_item:
+    label: 'Rendered HTML output'
+    property_path: rendered_item
     type: text
-    dependencies:
-      module:
-        - node
+    configuration:
+      roles:
+        - anonymous
```

Comment:
I'm pretty sure we have this covered through the logic that locks the whole site down, but noticing that this doesn't seem right.